packages/dart/npt_flutter/windows/runner/main.cpp

```diff
@@ -13,21 +15,27 @@ int APIENTRY wWinMain(_In_ HINSTANCE instance, _In_opt_ HINSTANCE prev,
     CreateAndAttachConsole();
   }
 
+  HWND hwnd = ::FindWindow(kWindowClassName, kWindowName);
+  if (hwnd != NULL) {
+    ::ShowWindow(hwnd, SW_NORMAL);
+    ::SetForegroundWindow(hwnd);
+    return EXIT_FAILURE;
+  }
+
```

Comment:
If we find a window matching the default Flutter window class name && Titled "NoPorts Desktop" open that instead.